blocks/algolia-assortment-content-source-block/sources/algolia-assortment.js

```diff
@@ -0,0 +1,49 @@
+import algoliasearch from "algoliasearch/lite";
+import { algoliaSearchKey, algoliaAppId } from "fusion:environment";
+
+const params = {
+	filters: "text",
+	index: "text",
+	query: "text",
+	ruleContexts: "text",
+	hitsPerPage: "number",
+	page: "number",
+};
+
+const fetch = async (contentSourceParameters) => {
+	const { filters, index, query, ruleContexts } = contentSourceParameters;
+	let { hitsPerPage, page } = contentSourceParameters;
+
+	// parameters in content source are defined as null by default
+	if (hitsPerPage === null) {
+		hitsPerPage = 20;
+	}
+	if (page === null) {
+		page = 0;
+	}
```

Comment:
Any reason not to set defaults like so ?

```suggestion
	const {
		filters,
		index,
		query,
		ruleContexts,
		hitsPerPage = 20,
		page = 0,
	} = contentSourceParameters;
```

Reply:
yes. in the comment and test, I said the values were coming in as null. So we can't use default parameters @matthewroach